sdk/core/azure-core-http-okhttp/pom.xml

```diff
@@ -90,6 +96,7 @@
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock-standalone</artifactId>
+      <version>2.24.1</version>
```

Comment:
Let's take this version out, and make sure parent is the same.

Reply:
yup, updated.